source/MRViewer/MRUIStyle.h

```diff
@@ -402,7 +402,7 @@ MRVIEWER_API void alignTextToButton( float scaling );
 
 /// Highlight the background of current window's bottom part, starting from current position
 /// Some padding is added
-MRVIEWER_API void highlightWindowBottom( float scaling );
+MRVIEWER_API void highlightWindowBottom( float scaling, ImVec2* min = nullptr, ImVec2* max = nullptr );
```

Comment:
why non const pointers? let it be const ImVec2& min = ImVec2(FLT_MAX,FLT_MAX) and max = -FLT_MAX,-FLT_MAX

Reply:
or suggest that if min < 0 then it is start of window, if max < min than it goes upto the end of window